scripts/setup.sh

```diff
@@ -39,9 +39,7 @@ function setup_client() {
 	yarn install
 }
 
-venv
-setup_server
-setup_client
+venv && setup_server && setup_client
 
-yarn start:dev & python manage.py runserver 0.0.0.0:8000
```

Comment:
There is a difference between using a single `&` and a double `&&` symbol. Do you know the difference? and was the former removed for a reason?

Reply:
Yes @andela-sjames ,& lets the other command run in the background while && waits for the first command to complete then execute the next. We didn't remove it, but rather moved to line 44.